configure-places-in-the-sdk/places-extension/using-places-without-active-region-monitoring.md

```diff
@@ -0,0 +1,415 @@
+2. [Retrieve nearby Points of Interest from the SDK based on the new location](#retrieve-nearby-points-of-interest-from-the-sdk)
+3. [Trigger entry events when the user is in a POI](#trigger-entry-events-when-the-user-is-in-a-poi)
+
+## Collect the User's Location
+
+The app developer is responsible for collecting the current location of the device. This can be done via `CoreLocation.framework` (iOS) or the `Location` APIs provided by the `Google Play Services` (Android).
+
+For more information, please see the documentation provided by Apple and Google:
+
+- [CoreLocation](https://developer.apple.com/documentation/corelocation)
+- [Location APIs in Google Play Services](https://developer.android.com/training/location)
+
+## Retrieve Nearby Points of Interest from the SDK
+
+Once you have obtained the user's location, you can pass it to the SDK to get back a list of nearby POIs.  
+
+{% tabs %}
+
+{% tab title="Android (Java)" %}
+
+Example implementation using a [`BroadcastReceiver`](https://codelabs.developers.google.com/codelabs/background-location-updates-android-o/index.html?index=..%2F..index#5) in Android:
+
+```java
+public class LocationBroadcastReceiver extends BroadcastReceiver {
+    static final String ACTION_LOCATION_UPDATE = "locationUpdate";
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        if (intent == null || context == null) {
+            return;
+        }
+
+        final String action = intent.getAction();
+        if (!ACTION_LOCATION_UPDATE.equals(action)) {
+            return;
+        }
+
+        LocationResult result = LocationResult.extractResult(intent);
+        if (result == null) {
+            return;
+        }
+
+        Location currentLocation = result.getLastLocation();
+        if (currentLocation == null) {
+            return;
+        }
+
+        // ask the Places SDK for the 10 nearest Points of Interest based on the user's location
+        Places.getNearbyPointsOfInterest(currentLocation, 10,
+                new AdobeCallback<List<PlacesPOI>>() {
+                    @Override
+                    public void call(List<PlacesPOI> pois) {
+                        // pois is the 10 nearest POIs based on the location
+                    }
+                }, new AdobeCallback<PlacesRequestError>() {
+                    @Override
+                    public void call(PlacesRequestError placesRequestError) {
+                        // Look for the placesRequestError and handle the error accordingly
+                    }
+                });
+    }
+}
+```
+{% endtab %}
+
+{% tab title="iOS (Objective-C)" %}
+
+Example implementation from a [`CLLocationManagerDelegate`](https://developer.apple.com/documentation/corelocation/cllocationmanager?language=objc) method [`locationManager:didUpdateLocations:`](https://developer.apple.com/documentation/corelocation/cllocationmanagerdelegate/1423615-locationmanager?language=objc):
```

Comment:
Please reword to:
"Here is a sample implementation from a [`CLLocationManagerDelegate`](https://developer.apple.com/documentation/corelocation/cllocationmanager?language=objc) method [`locationManager:didUpdateLocations:`](https://developer.apple.com/documentation/corelocation/cllocationmanagerdelegate/1423615-locationmanager?language=objc):"